### DISCUSSION TITLE
RTL sample rate

### DISCUSSION THREAD

**Discussion by habub** [merbanan/rtl_433#2194] [Q&A] [answered]
I can't set any sample rate below the default of 250 KHz. I always get the same error:
_Invalid sample rate: 50000 Hz
WARNING: Failed to set sample rate._

Is this a known limitation or issue? Anyway arround it?

**Comment by merbanan** [comment, marked as answer]
225kHz is the minimum the device can support. And there is no way around it.

**Comment by zuckschwerdt** [comment]
For ref: https://github.com/osmocom/rtl-sdr/blob/5e73f90f1d85d8db2e583f3dbf1cff052d71d59b/src/librtlsdr.c#L1098-L1099
and also in librtlsdr: https://github.com/librtlsdr/librtlsdr/blob/710d578901a08b091748d4fd4471e3327a4d099e/src/librtlsdr.c#L2102-L2103
so valid rates are 225k to 300k and 900k to 3200k, the usual rates used are 250k, 1000k, 1024k and maybe 2048k.
